### DISCUSSION TITLE
"dev"-property in nuxt3

### DISCUSSION THREAD

**Discussion by holozaen** [nuxt/nuxt#16128] [Questions] [answered]
Hello
In nuxt2 there has been a "dev"-property to check at runtime whether the app runs in development or production mode.
How can I access this information in nuxt3?
Thank you very much for your help.
Best Regards, 
Peter

**Comment by TheAlexLichter** [comment, marked as answer]
You can still use `process.dev` as before.

Also, you might want to check out `<DevOnly>` components, see e.g. - https://github.com/nuxt/framework/pull/7950/files#diff-b8a17e77ffacc1bab630818f83acf51a57d9738356baff2bb3413e56f2f8b665 and [the docs](https://nuxt.com/docs/guide/directory-structure/components#devonly-component)

**Reply by holozaen** [reply]
Ah. Great answer. Thank you!
